package.json

```
@@ -47,24 +47,24 @@
     ]
   },
   "dependencies": {
-    "stylelint-config-recommended": "^11.0.0"
+    "stylelint-config-recommended": "^12.0.0"
   },
   "devDependencies": {
     "@stylelint/prettier-config": "^2.0.0",
     "@stylelint/remark-preset": "^4.0.0",
-    "eslint": "^8.36.0",
+    "eslint": "^8.38.0",
     "eslint-config-stylelint": "^18.0.0",
     "eslint-plugin-jest": "^27.2.1",
     "husky": "^8.0.3",
     "jest": "^29.5.0",
-    "lint-staged": "^13.2.0",
-    "np": "^7.6.3",
+    "lint-staged": "^13.2.1",
+    "np": "^7.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.8.4",
+    "prettier": "^2.8.7",
     "remark-cli": "^11.0.0",
-    "stylelint": "^15.4.0"
+    "stylelint": "stylelint/stylelint"
```

- [ ] We will update `stylelint` to `15.5.0` before the merge. See https://github.com/stylelint/stylelint/issues/6780

```suggestion
    "stylelint": "^15.5.0"
```